<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_UI</class>
 <widget class="QDialog" name="login_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LOGIN IN</string>
  </property>
  <property name="windowIcon">
   <iconset resource="backgroundPNG.qrc">
    <normaloff>:/background/background/background.jpg</normaloff>:/background/background/background.jpg</iconset>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#login_UI{
	
	background-image: url(:/background/background/42.png); /* 背景图片路径 */
background-position: center; /* 将背景图片放置在组件的中心位置 */
background-repeat: no-repeat; /* 禁止背景图片重复 */
background-attachment: fixed; 
}

QLineEdit{
	background-color: rgb(62, 62, 62); 
	color: white;
}

QPushButton{
	background-color: rgb(62, 62, 62); 
	color: white;
}</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="5,0,0,0">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_username">
           <property name="palette">
            <palette>
             <active/>
             <inactive/>
             <disabled/>
            </palette>
           </property>
           <property name="font">
            <font>
             <family>华文细黑</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>true</bold>
             <underline>true</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>username :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>le_username</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_passwd">
           <property name="palette">
            <palette>
             <active/>
             <inactive/>
             <disabled/>
            </palette>
           </property>
           <property name="font">
            <font>
             <family>华文细黑</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>true</bold>
             <underline>true</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>password :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>le_passwd</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="le_username">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>CrossCursor</cursorShape>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_passwd">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>CrossCursor</cursorShape>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>60</number>
      </property>
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="pb_login_in">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Login in</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_sign_up">
        <property name="cursor">
         <cursorShape>OpenHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Sign up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_randPasswd">
        <property name="text">
         <string>生成随机密码</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_tips">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>49</red>
           <green>147</green>
           <blue>147</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>49</red>
           <green>147</green>
           <blue>147</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>49</red>
           <green>147</green>
           <blue>147</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>49</red>
           <green>147</green>
           <blue>147</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>49</red>
           <green>147</green>
           <blue>147</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>49</red>
           <green>147</green>
           <blue>147</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>49</red>
           <green>147</green>
           <blue>147</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>49</red>
           <green>147</green>
           <blue>147</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>49</red>
           <green>147</green>
           <blue>147</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>方正姚体</family>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#label_tips{
	
	background-color: rgb(49, 147, 147);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>*** *** 等待连接服务器 *** ***</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_username</tabstop>
  <tabstop>le_passwd</tabstop>
  <tabstop>pb_login_in</tabstop>
  <tabstop>pb_sign_up</tabstop>
 </tabstops>
 <resources>
  <include location="backgroundPNG.qrc"/>
 </resources>
 <connections/>
</ui>
